common/Resources/ti.main.js

```diff
@@ -40,4 +40,9 @@ require('./ti.internal/bootstrap.loader').loadAsync(function () {
 	// We've finished loading/executing all bootstrap scripts.
 	// We can now proceed to run the main "app.js" script.
 	require('./app');
+
+	// This event is to be fired after "app.js" execution. Reasons:
+	// - Allow system to queue startup related events until "app.js" has had a chance to add listeners.
+	// - For Alloy apps, we now know that Alloy has been initialized and its globals were added.
+	Ti.App.fireEvent('started');
```

Comment:
Do we want to document this event? If this is an internal event maybe rename to `_started` ?

Reply:
I don't have a problem publicly documenting it.

@sgtcoolguy, do you have any objections? The above adds a new `Ti.App` "started" event on all platforms (Android, iOS, and Windows). It is fired after "app.js" gets executed. For example, this is useful for deferring startup related events until after the "app.js" has had a chance to add listeners for them.